translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+"3.\n"
+"\t\tkubectl scale —current-replicas=2 —replicas=3 deployment/mysql\n"
+"\n"
+"\t\t# Escala múltiplos replication controllers.\n"
+"\t\tkubectl scale —replicas=5 rc/foo rc/bar rc/baz\n"
+"\n"
+"\t\t# Escala um Job chamado ‘cron’ to 3.\n"
+"\t\tkubectl scale —replicas=3 job/cron"
+
+#: pkg/kubectl/cmd/apply_set_last_applied.go:67
+msgid ""
+"\n"
+"\t\t# Set the last-applied-configuration of a resource to match the contents "
+"of a file.\n"
+"\t\tkubectl apply set-last-applied -f deploy.yaml\n"
+"\n"
+"\t\t# Execute set-last-applied against each configuration file in a "
+"directory.\n"
+"\t\tkubectl apply set-last-applied -f path/\n"
+"\n"
+"\t\t# Set the last-applied-configuration of a resource to match the contents "
+"of a file, will create the annotation if it does not already exist.\n"
+"\t\tkubectl apply set-last-applied -f deploy.yaml --create-annotation=true\n"
+"\t\t"
+msgstr ""
+"\n"
+"\t\t# Ajusta o last-applied-configuration de um recurso para corresponder ao "
+"conteúdo de um arquivo.\n"
+"\t\tkubectl apply set-last-applied -f deploy.yaml\n"
+"\n"
+"\t\t# Executa o set-last-applied em todas os arquivos de configuração no "
+"diretório.\n"
+"\t\tkubectl apply set-last-applied -f path/\n"
+"\n"
+"\t\t# Ajusta o last-applied-configuration de um recurso para corresponder ao "
+"conteúdo de um arquivo, será criada uma annotation se esta ainda não "
+"existe.\n"
+"\t\tkubectl apply set-last-applied -f deploy.yaml —create-annotation=true\n"
+"\t\t"
+
+#: pkg/kubectl/cmd/top_pod.go:61
+msgid ""
+"\n"
+"\t\t# Show metrics for all pods in the default namespace\n"
+"\t\tkubectl top pod\n"
+"\n"
+"\t\t# Show metrics for all pods in the given namespace\n"
+"\t\tkubectl top pod --namespace=NAMESPACE\n"
+"\n"
+"\t\t# Show metrics for a given pod and its containers\n"
+"\t\tkubectl top pod POD_NAME --containers\n"
+"\n"
+"\t\t# Show metrics for the pods defined by label name=myLabel\n"
+"\t\tkubectl top pod -l name=myLabel"
+msgstr ""
+"\n"
+"\t\t# Mostra as métricas para todos os pods no namespace default\n"
+"\t\tkubectl top pod\n"
+"\n"
+"\t\t# Mostra as métricas para todos os pods em um dado namespace\n"
+"\t\tkubectl top pod —namespace=NAMESPACE\n"
+"\n"
+"\t\t# Mostra as métricas para um dado pod e seus containers\n"
+"\t\tkubectl top pod POD_NAME —containers\n"
+"\n"
+"\t\t# Mostra as métricas para os pods definidos pelo label name=myLabel\n"
+"\t\tkubectl top pod -l name=myLabel"
+
+#: pkg/kubectl/cmd/stop.go:40
+msgid ""
+"\n"
+"\t\t# Shut down foo.\n"
+"\t\tkubectl stop replicationcontroller foo\n"
+"\n"
+"\t\t# Stop pods and services with label name=myLabel.\n"
+"\t\tkubectl stop pods,services -l name=myLabel\n"
+"\n"
+"\t\t# Shut down the service defined in service.json\n"
+"\t\tkubectl stop -f service.json\n"
+"\n"
+"\t\t# Shut down all resources in the path/to/resources directory\n"
+"\t\tkubectl stop -f path/to/resources"
+msgstr ""
+"\n"
+"\t\t# Termina o foo.\n"
```

Comment:
I think here in the original text they should have used "Shut down the replicationcontroller foo", so maybe here this can be "Termina o replicationcontroller foo"